lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -305,7 +305,7 @@ define((require, exports, module) ->
       reasons = []
       if not start.correct
         reasons.push(start.reason)
-      if not end.correct
+      if not end.correct and not (reasons[0]? and reasons[0] == end.reason)
```

Comment:
Why do you always compare with  [0] ? Can the duplicate be at [1] ?

Reply:
At this point in code, `reasons` can have only one or zero elements, because it is created just 2 lines above.